src/lib.rs

```diff
@@ -1,8 +1 @@
-// The typed-html crate does pretty deep macro calls. Bump if
-// recursion limit compilation errors return for html!() calls.
-#![recursion_limit = "256"]
```

Comment:
Are you sure we don't need this?

Reply:
I think he removed the dashboard entirely in previous prs

Reply:
Then I guess we should remove the package from cargo if it's not used anymore.